stable/redis-ha/templates/redis-haproxy-service.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.haproxy.enabled }}
+apiVersion: v1
```

Comment:
Could you add a test for this service? Likely will need to add test values to enable the HAproxy
https://github.com/helm/charts/tree/master/test#providing-custom-test-values

Reply:
@ssalaues Done, can you take a look?